### PR TITLE
Create Awesome Books Project Week 6

### PR DESCRIPTION
**Pull request by calebchris000**
## The following features were added:
- Create a collection that keeps a list of books.
- Create a function to add a new book to the collection, with title and author.
- Create a function to remove a book from the collection.
- Make sure that data is preserved in the browser's memory.

### CODE REVIEW

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi @calebchris000 and @ngeelani48  

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that doesn't have a professional readme kindly add a professional readme file


Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: app.js [hunk 1]
```diff
@@ -1,54 +1,73 @@
...
+// Function that removes the data when the user presses the remove button
+function createRemove(text, parent) {
+  const removeButton = document.createElement('button');
+  removeButton.textContent = text;
+  removeButton.addEventListener('click', () => {
+    bookshelf.removeChild(parent);
+    localStorage.removeItem('data');
+  });
+  parent.appendChild(removeButton);
+}
```
Comment (thecodechaser) [inline app.js]: - [ ] When I'm trying to remove the added books, The books don't remove from the collection. I checked there's no error on the console, I think the event is not being fired. Please make sure to fix this issue and users should be able to remove the added books from the collection.

**Review by thecodechaser — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ 

## Hi @ngeelani48, and @calebchris000👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
- Highly organized code. ✔️
- No linter errors. :heavy_check_mark:
- GitHub flow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app.js (hunk 1 above)
Comment (calebchris000) [reply app.js]: @thecodechaser, thank you for the feedback. However, I don't understand what the issue is. If you are talking about the books removing from the collection, it is actually being removed. Can you please clarify?

Comment (topeogunleye) [inline app.js]: - [ ] As suggested by the previous reviewer, kindly work on your remove functionality, when I remove a book, all books gets removed from the local storage and when I reload the page all books get removed from the user interface as well. Kindly check my [loom video](https://www.loom.com/share/a1e6bede646846cc935d5f0649a506c7) here for more explanation:


https://www.loom.com/share/a1e6bede646846cc935d5f0649a506c7

**Review by topeogunleye — CHANGES_REQUESTED** [review]
Hi @ngeelani48, 

Please work on the changes requested by the previous reviewer. 👏 


 ## Highlights
* Good commit messages. :heavy_check_mark:
* No linter errors. :heavy_check_mark:
* Followed Github flow :heavy_check_mark:
* Professional README file. :heavy_check_mark:
* PR has a good title and summary :heavy_check_mark:

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

You can also consider:
- Nothing to mention.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app.js (hunk 1 above)
Comment (calebchris000) [reply app.js]: Thank you for the clarification. I will fix that right away.

**Pull request closed without merging**